lib/OrganizationsFieldArray/OrganizationsFieldArray.js

```diff
@@ -11,74 +11,75 @@ import OrganizationPicker from './OrganizationPicker';
-      isEmptyMessage: PropTypes.node,
-      items: PropTypes.arrayOf(PropTypes.object),
-      name: PropTypes.string.isRequired,
-      onAddField: PropTypes.func.isRequired,
-      onDeleteField: PropTypes.func.isRequired,
-      onMarkForDeletion: PropTypes.func.isRequired,
-      onReplaceField: PropTypes.func.isRequired,
-      organizationPickerComponent: PropTypes.elementType,
-      roles: PropTypes.arrayOf(PropTypes.object),
-      uniqueRole: PropTypes.string,
-    };
+  static propTypes = {
+    addOrganizationBtnLabel: PropTypes.node,
+    isEmptyMessage: PropTypes.node,
+    items: PropTypes.arrayOf(PropTypes.object),
+    name: PropTypes.string.isRequired,
+    onAddField: PropTypes.func.isRequired,
+    onDeleteField: PropTypes.func.isRequired,
+    onMarkForDeletion: PropTypes.func.isRequired,
+    onReplaceField: PropTypes.func.isRequired,
+    organizationPickerComponent: PropTypes.elementType,
+    roles: PropTypes.arrayOf(PropTypes.object),
+    uniqueRole: PropTypes.string,
+  };
 
-    static defaultProps = {
-      addOrganizationBtnLabel: <FormattedMessage id="stripes-erm-components.organizations.addOrganization" />,
-      isEmptyMessage: <FormattedMessage id="stripes-erm-components.organizations.noOrganizations" />,
-      organizationPickerComponent: OrganizationPicker,
-    }
+  static defaultProps = {
+    addOrganizationBtnLabel: <FormattedMessage id="stripes-erm-components.organizations.addOrganization" />,
+    isEmptyMessage: <FormattedMessage id="stripes-erm-components.organizations.noOrganizations" />,
+    organizationPickerComponent: OrganizationPicker,
+  }
 
-    renderEmpty = () => (
-      <Layout className="padding-bottom-gutter" data-test-org-empty-message>
-        {this.props.isEmptyMessage}
-      </Layout>
-    );
-
-    renderOrganizations = (items) => {
-      const orgs = items.map((organization, index) => {
-        return (
-          <EditOrganizationCard
-            index={index}
-            key={index}
-            name={this.props.name}
-            onDeleteField={this.props.onDeleteField}
-            onMarkForDeletion={this.props.onMarkForDeletion}
-            onReplaceField={this.props.onReplaceField}
-            organization={organization}
-            organizationPickerComponent={this.props.organizationPickerComponent}
-            roles={this.props.roles}
-            uniqueRole={this.props.uniqueRole}
-          />
-        );
-      });
+  renderEmpty = () => (
+    <Layout className="padding-bottom-gutter" data-test-org-empty-message>
+      {this.props.isEmptyMessage}
+    </Layout>
+  );
 
+  renderOrganizations = (items) => {
+    const orgs = items.map((organization, index) => {
       return (
-        <div>
-          {orgs}
-        </div>
+        <EditOrganizationCard
+          index={index}
+          key={index}
+          deleteButtonTooltipText={<FormattedMessage id="stripes-erm-components.organizations.removeOrganization" />}
```

Comment:
This FormattedMessage would preferably just be defined in the card component itself since it doesn't look like we'll need to customise it from outside of that component.